tests/scoring/test_data_links.py

```diff
@@ -1,7 +1,14 @@
 from nplinker.scoring.linking.data_linking import DataLinks
-from .test_metabolomics import spec_dict, spec_with_families
-from .test_gnps_molecular_family_loader import molecular_families_gnps
+from nplinker.metabolomics.gnps.gnps_molecular_family_loader import GNPSMolecularFamilyLoader
+from .. import DATA_DIR
+import os
+import pytest
 
+@pytest.fixture
```

Comment:
Maybe move this also to conftest?

Reply:
it's only used by this file, so better to keep it here. `conftest.py` is a place for fixtures shared by different modules/files.